privacyidea/lib/token.py

```diff
@@ -1081,8 +1076,7 @@ def init_token(param, user=None, tokenrealms=None,
     if user is not None and user.login != "":
         tokenobject.add_user(user)
 
-    upd_params = param
-    tokenobject.update(upd_params)
+    tokenobject.update(param)
```

Comment:
Im am not sure anymore, if there is a reason, why the dict was copied - if it was necessary for a deep copy since it gets modified somewhere.

Reply:
As far as I can tell, this does not make any difference. Because `upd_params = param` does not copy the dict, but only assigns a second name for reference. And python makes calls with 'pass by object-reference', so if `tokenobject.update()` changes `param` or `upd_params` would not make a difference because both reference the same dictionary.